api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -435,6 +436,20 @@ public void testSetBypassTimes() {
 
   @Test
   public void testSubmitTermsOfService() {
+    // Testing NULL input version
```

Comment:
suggest to put this in a new test method, something like `testSubmitTermsOfService_invalidVersion()`
We usually prefer each test method focus on one scenario, and keep it simple